sdk/helper/testcluster/docker/environment.go

```diff
@@ -773,6 +773,48 @@ func (n *DockerClusterNode) Start(ctx context.Context, opts *DockerClusterOption
 	return nil
 }
 
+func (n *DockerClusterNode) Pause(ctx context.Context) error {
+	return n.DockerAPI.ContainerPause(ctx, n.Container.ID)
+}
+
+func (n *DockerClusterNode) AddNetworkDelay(ctx context.Context, delay time.Duration, targetIP string) error {
```

Comment:
Is there a scenario for which we may need to revert the network delay within the same test? I understand that this is going to be used only for docker, and the benefit of it is to remove the docker image on the test tear down. I am just curious!